modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -78,6 +78,40 @@
 	///list containing cooking radial buttons, for when anything of /obj/item/food is used on the forge
 	var/static/list/radial_options = list("oven" = radial_oven, "microwave" = radial_microwave)
 
+    ///Blacklist that contains reagents that weapons and armor are unable to be imbued with.
+	var/static/list/disallowed_reagents = typecacheof(list(
+		/datum/reagent/inverse/,
+		/datum/reagent/consumable/entpoly,
+		/datum/reagent/pax,
+		/datum/reagent/consumable/liquidelectricity/enriched,
+		/datum/reagent/toxin/heparin,
+		/datum/reagent/teslium,
+		/datum/reagent/eigenstate,
+		/datum/reagent/drug/bath_salts,
+		/datum/reagent/drug/methamphetamine,
+		/datum/reagent/drug/pcp,
+		/datum/reagent/consumable/cum,
+		/datum/reagent/consumable/femcum,
+		/datum/reagent/consumable/breast_milk,
+		/datum/reagent/toxin/chloralhydrate,
+		/datum/reagent/toxin/bonehurtingjuice,
+		/datum/reagent/toxin/lexorin,
+		/datum/reagent/toxin/staminatoxin,
+		/datum/reagent/toxin/sulfonal,
+		/datum/reagent/toxin/histamine,
+		/datum/reagent/medicine/atropine,
+		/datum/reagent/toxin/fentanyl,
+		/datum/reagent/toxin/acid,
+		/datum/reagent/phlogiston,
+		/datum/reagent/napalm,
+		/datum/reagent/thermite,
+		/datum/reagent/medicine/earthsblood,
+		/datum/reagent/medicine/ephedrine,
+		/datum/reagent/medicine/epinephrine,
+		/datum/reagent/medicine/synaptizine,
+		/datum/reagent/medicine/c2/penthrite,
```

Comment:
remove these-- medicines allow people to create heal staves and other misc. items.